_translator-files/po/de/Contribution.po

```diff
@@ -43,38 +43,32 @@ msgstr "Beitragen"
 
 #. type: Title #
 #: ../wiki/en/Contribution.md:8
-#, fuzzy, no-wrap
-#| msgid "# Contributing to the Jamulus Project"
+#, no-wrap
 msgid "Contributing to the Jamulus Project"
-msgstr "#  Zum Jamulus-Projekt beitragen"
+msgstr "Zum Jamulus-Projekt beitragen"
 
 #. type: Plain text
 #: ../wiki/en/Contribution.md:11
-#, fuzzy, no-wrap
-#| msgid "If you find a mistake, typo or something out of date (in any language) on this website or in Jamulus itself, you can <a href=\"https://github.com/jamulussoftware/jamulus/issues\">raise it as a bug</a>."
+#, no-wrap
 msgid "If you find a mistake, typo or something out of date (in any language) on this website or in Jamulus itself, you can <a href=\"https://github.com/jamulussoftware/jamulus/issues\">raise it as a bug</a>.\n"
-msgstr "Wenn du auf dieser Website oder in Jamulus selbst einen Fehler, einen Tippfehler oder etwas Veraltetes (egal in welcher Sprache) findest, kannst du <a href=\"https://github.com/jamulussoftware/jamulus/issues\">Verbesserungen hier melden</a>."
+msgstr "Wenn du auf dieser Website oder in Jamulus selbst einen Fehler, einen Tippfehler oder etwas Veraltetes (egal in welcher Sprache) findest, kannst du <a href=\"https://github.com/jamulussoftware/jamulus/issues\">Verbesserungen hier melden</a>.\n"
 
 #. type: Plain text
 #: ../wiki/en/Contribution.md:13
-#, fuzzy, no-wrap
-#| msgid "If you think some documentation or information is missing or can be improved, <a href=\"https://github.com/jamulussoftware/jamulus/discussions\">post about it on this forum</a> so it can be discussed first."
+#, no-wrap
 msgid "If you think some documentation or information is missing or can be improved, <a href=\"https://github.com/jamulussoftware/jamulus/discussions\">post about it on this forum</a> so it can be discussed first.\n"
-msgstr "Wenn du der Meinung bist, dass eine Dokumentation oder eine Information fehlt oder verbessert werden kann, <a href=\"https://github.com/jamulussoftware/jamulus/discussions\">poste deine Meinung in diesem Forum</a>, damit es zuerst diskutiert werden kann."
+msgstr "Wenn du der Meinung bist, dass eine Dokumentation oder eine Information fehlt oder verbessert werden kann, <a href=\"https://github.com/jamulussoftware/jamulus/discussions\">poste deine Meinung in diesem Forum</a>, damit es zuerst diskutiert werden kann.\n"
```

Comment:
Needs rewording. Eine Dokumentation oder eine Information fehlt is not valid German.